instrumentation/dubbo-rpc/src/main/java/brave/dubbo/rpc/DubboParser.java

```diff
@@ -69,19 +69,28 @@ static boolean parseRemoteIpAndPort(Span span) {
   }
 
   /**
-   * We decided to not map Dubbo codes to human readable names like {@link
-   * RpcException#BIZ_EXCEPTION} even though we defined "rpc.error_code" as a human readable name.
-   *
-   * <p>The reason was a comparison with HTTP status codes, and the choice was between returning
-   * just numbers or reusing "UNKNOWN_EXCEPTION" which is defined in Dubbo for code "0" for any
-   * unknown code. Returning numbers was the less bad option as it doesn't conflate code words.
-   *
-   * <p>Later, we can revert this back to code words, but once this gets into the RPC mapping for
-   * Dubbo it will be hard to change.
+   * This library is no-longer being released, so it should not have any maintenance on error codes.
+   * The error codes here were defined in 2012.
    */
   @Nullable static String errorCode(Throwable error) {
     if (error instanceof RpcException) {
-      return String.valueOf(((RpcException) error).getCode());
+      int code = ((RpcException) error).getCode();
+      switch (code) {
+        case RpcException.UNKNOWN_EXCEPTION:
+          return "UNKNOWN_EXCEPTION";
+        case RpcException.NETWORK_EXCEPTION:
+          return "NETWORK_EXCEPTION";
+        case RpcException.TIMEOUT_EXCEPTION:
+          return "TIMEOUT_EXCEPTION";
+        case RpcException.BIZ_EXCEPTION:
+          return "BIZ_EXCEPTION";
+        case RpcException.FORBIDDEN_EXCEPTION:
+          return "FORBIDDEN_EXCEPTION";
+        case RpcException.SERIALIZATION_EXCEPTION:
+          return "SERIALIZATION_EXCEPTION";
+        default:
+          return String.valueOf(code);
```

Comment:
I get where this is coming from. Just to share only one point I'm concerned about - if some servers in a fleet use a new version of instrumentation and others don't, the same exception shows up as name in some, int in others. It does seem quite rare though since why not bump instrumentation when bumping Dubbo? So all good here.

Reply:
the only other option I can think of is using reflection in to lookup the symbols. Ex search for public static ints whose name ends in _EXCEPTION

that would make it no maintenance and since they are all public, unlikely to cause a weird crash.. wdyt?

Reply:
I will give it a try. we use one-time reflection in a lot of code, like servlet etc. we would have heard complaints by now if someone is unable to use reflection at all..

Reply:
Thanks, that sounds like a nice approach!

Reply:
couldn't use _EXCEPTION as they are not consistent :) we blindly trust int constants for now. It is unlikely to cause an issue I think and I agree this is better. thanks, rag!